### PR TITLE
Eric

### PR DESCRIPTION
**Pull request by moro-patana**
Hey, 

I am relly sorry, I could find melky's biography so I created new branch.

Please check my work .
Thank you,
Manjaka

### CODE REVIEW

File: e/eric.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Eric Manana
+
+
+![Picture of eric](eric.jpg)
+
+Erick Manana is an acoustic guitarist, singer and songwriter from Madagascar. He often performs in a duo or as a solo artist, singing in accompaniment to his acoustic guitar in the ba-gasy genre that gained prominence in the central highlands of Madagascar in the 1930s. Wikipedia
```
Comment (ndorfin) [inline e/eric.md]: You can remove the `wikipedia` bit at the end

File: e/eric.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+Born: 1959 (age 61 years), Madagascar
+
+
+
+**Genre:** Salegy
+
+**Artist homepage:** [Eric](https://www.google.com/search?q=erick+manana&sxsrf=ACYBGNTd-BamXIWo-a9lf5gztAZwTUuDuA:1580378860880&source=lnms&sa=X&ved=0ahUKEwiZ1vOEiqvnAhU68uAKHW_IC1QQ_AUIDSgA&biw=1366&bih=576&dpr=1)
```
Comment (ndorfin) [inline e/eric.md]: Can you find a homepage for this artist? If not, remove this line.

File: e/eric.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Eric Manana
```
Comment (ndorfin) [inline e/eric.md]: If this artist is called `Eric Manana` and not `Eric`, then we should update the filenames to be `eric-manana` and we fix the link in `e/index.md`

**Review by ndorfin — CHANGES_REQUESTED** [review]

File: e/eric.md (hunk 3 above)
Comment (moro-patana) [reply e/eric.md]: I fixed it so please check it.

Thank you,

File: e/eric-manana.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+
+
+Erick Manana is an acoustic guitarist, singer and songwriter from Madagascar. He often performs in a duo or as a solo artist, singing in accompaniment to his acoustic guitar in the ba-gasy genre that gained prominence in the central highlands of Madagascar in the 1930s.
+Born: 1959 (age 61 years), Madagascar
+
+**Genre:** Salegy
+
+
+
```
Comment (ndorfin) [inline e/eric-manana.md]: Remove these extra lines

**Review by ndorfin — CHANGES_REQUESTED** [review]

File: e/eric-manana.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
+# Eric Manana
+
+![Picture of eric](eric-manana.jpg
```
Comment (ndorfin) [inline e/eric-manana.md]: This image tag looks broken

**Review by ndorfin — CHANGES_REQUESTED** [review]
You can delete the two other files too.

File: e/eric-manana.md (hunk 5 above)
Comment (moro-patana) [reply e/eric-manana.md]: I have fixed the image.

**Review by ndorfin — APPROVED** [review]
Nice one

**Pull request merged**